rllib/evaluation/rollout_worker.py

```diff
@@ -857,43 +653,16 @@ def wrap(env):
             )
         )
 
-    @DeveloperAPI
+    @override(EnvRunner)
     def assert_healthy(self):
-        """Checks that __init__ has been completed properly.
-
-        Useful in case a RolloutWorker is run as @ray.remote (Actor) and the owner
-        would like to make sure the worker has been properly initialized.
-
-        Returns:
-            True if the worker is properly initialized
-        """
         is_healthy = self.policy_map and self.input_reader and self.output_writer
         assert is_healthy, (
             f"RolloutWorker {self} (idx={self.worker_index}; "
             f"num_workers={self.num_workers}) not healthy!"
         )
 
-    @DeveloperAPI
+    @override(EnvRunner)
     def sample(self) -> SampleBatchType:
-        """Returns a batch of experience sampled from this worker.
```

Comment:
I think we need to keep the docstring around, right?

Reply:
Maybe in this case, yes. I was thinking since I moved it into the base class that I wouldn't need this here. But the behavior might be different still.

Reply:
fixed.